changelog.d/1554.change.rst

```diff
@@ -0,0 +1 @@
+``build_meta.build_sdist`` includes the *setup.py* file by default for scenarios where the invoking script is not "setup.py".
```

Comment:
This should be always the case, maybe just say that build sdist ensures that setup.py is always and automatically packaged.